src/context/viewer-context.tsx

```diff
@@ -100,15 +130,31 @@ export interface ViewerAction {
 export const defaultState: ViewerContextStore = {
   activeCanvas: "",
   activeManifest: "",
+  autoScroll: expandAutoScrollOptions(
+    defaultConfigOptions?.informationPanel?.autoScroll,
+  ),
```

Comment:
Same, dup. Or perhaps I'm not fully understanding the purpose of putting this here as well as on the `configOption.informationPanel.vtt.autoScroll`.

Reply:
Another reason for separating the `configOptions` scroll setting from the top-level scroll setting in the state is to allow for `true` and `false` config values that automatically expand themselves to `{ enabled: [true|false], settings: DefaultSettings }`.